visual-e2e/examples/test/configs/wdio.screener.saucelabs.conf.js

```diff
@@ -14,8 +12,8 @@ config.capabilities = [
         platformName: 'macOS 10.15',
         browserVersion: 'latest',
         'sauce:options': {
-            username: config.user,
-            accesskey: config.accessKey
+            username: process.env.SAUCE_USERNAME,
+            accesskey: process.env.SAUCE_ACCESS_KEY
         },
         'sauce:visual': {
             apiKey: process.env.SCREENER_API_KEY,
```

Comment:
There is still Screener here in the env variable

Reply:
@wswebcreation this is what the documentation recommends